transcendence-app/migrations/20230109194454_add_2fa_secret_column_to_users_table.ts

```diff
@@ -0,0 +1,15 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.raw(`
+    ALTER TABLE Users
+      ADD COLUMN "twoFactorAuthenticationSecret" TEXT
```

Comment:
That's not how migrations work. Once you create a migration, you don't change it, ever. You can stack changes in additional migrations, but the past migrations stay the way they are. That's why they have the timestamp in the filename.
That's why they're powerful, but it's also why they're having, and why I asked to scrap them: to have one table scheme in one file. Well also have to scrap the database the way it currently is. Not a huge loss IMHO

Reply:
Thank you for the explanation of how the migration work, @Damrod.

Should we keep the migrations or scrape them?

One way to see the details of a table is using the `\d` command in `psql`, for example, to get the details of the `Users` table:

```
\d Users
```

Reply:
Yes, I was just thinking that having the tables in a file is always handier than having to go to the deployed service. But for me is ok, I though migrations worked differently. Thanks both of you!

Reply:
I'd like to let you know what do I think:
I like migrations, and I'm ready to keep them, but not because they allow us to preserve the database (we are before an alpha release, nobody expects us to have a consistent database), but because it makes it easy to inject data into the database at "build time".

If it was up to me, I'd just do a `docker volume prune` every time before running the app, and edit old migrations. It will make them backwards incompatible, but at least we'll be able to have "pretty migrations".